jobs/taar_lite_guidguid.py

```diff
@@ -94,18 +77,47 @@ def get_addons_per_client(broadcast_amo_whitelist, users_df):
     return df
 
 
+def get_table(view):
+    """Helper for determining what table underlies a user-facing view, since the Storage API can't read views."""
+    bq = bigquery.Client()
+    view = view.replace(":", ".")
+    # partition filter is required, so try a couple options
+    for partition_column in ["DATE(submission_timestamp)", "submission_date"]:
+        try:
+            job = bq.query(
+                f"SELECT * FROM `{view}` WHERE {partition_column} = CURRENT_DATE",
+                bigquery.QueryJobConfig(dry_run=True),
+            )
+            break
+        except Exception:
+            continue
```

Comment:
log some error

Reply:
it was copied from official telemetry docs https://docs.telemetry.mozilla.org/tools/spark.html#storage-api. I can add logging, but probably they don't consider an exception here as an error.

Reply:
added warning